andela/ah-frontend-sparta
# #164798187 User Should view their reading stats

#### What does this PR do?
Enable user to view their most recently viewed articles

#### Description of Task to be completed?
- Allow a user to view their reading stats

#### How should this be manually tested?
- Fetch this branch git fetch origin `users-should-see-their-reading-stats-164798187`
- Checkout to the branch git checkout `users-should-see-their-reading-stats-164798187`
- Run npm start to view the landing page
- Login to the app on `/Login`
- click `Check the Articles Reading Stats`

#### What are the relevant pivotal tracker stories?
[#164798187](https://www.pivotaltracker.com/story/show/164798187)

#### Screenshots (if appropriate)
<img width="1365" alt="Screenshot 2019-05-24 at 11 16 39" src="https://user-images.githubusercontent.com/35653907/58313143-8c040f00-7e15-11e9-98c5-7f392cf49839.png">

<img width="1263" alt="Screenshot 2019-05-23 at 22 00 25" src="https://user-images.githubusercontent.com/35653907/58279234-87ecd880-7da6-11e9-8b56-f355a71500e5.png">
